README.md

```diff
@@ -24,5 +24,7 @@ brew install exonum-java
 The formulae are licensed under the Apache License, Version 2.0.
 See [LICENSE](LICENSE) for details.
 
+[`RocksDB5`](Formula/rocksdb5.rb) formula is licensed under BSD 2-Clause License. See [BSD LICENSE](LICENSE-BSD.txt) for details.
```

Comment:
BSD does not seem to restrict re-licensing given the conditions are met (if we need it — we can relicense it under Apache 2).

Reply:
I'm not much experienced in licenses. Perhaps will give a question to legal.